golos.publication/golos.publication.cpp

```diff
@@ -767,9 +768,9 @@ void publication::send_poolerase_event(const structures::rewardpool& pool) {
     eosio::event(_self, "poolerase"_n, pool.created).send();
 }
 
-void publication::send_poststate_event(name author, const structures::message& post, base_t sharesfn) {
-    structures::post_event data{author, post.permlink, post.state.netshares, post.state.voteshares,
-        post.state.sumcuratorsw, sharesfn};
+void publication::send_poststate_event(name author, const structures::message& post, fixp_t sharesfn) {
+    structures::post_event data{ author, post.permlink, post.state.netshares, post.state.voteshares,
+        post.state.sumcuratorsw, static_cast<wide_t>(sharesfn) };
```

Comment:
@kaynarov check this cast, please. sharesfn in postevent and rsharesfn from poolstate must be in same units of measure.

Reply:
static_cast<wide_t>(sharesfn.data())
btw, why can't we just send (64 bits) sharesfn.data() here?